lucet-module/src/module_data.rs

```diff
@@ -58,6 +58,7 @@ pub struct ModuleFeatures {
     pub bmi2: bool,
     pub lzcnt: bool,
     pub popcnt: bool,
+    pub icnt: bool,
```

Comment:
`lzcnt` and `popcnt` only have the abbreviated names to match [CPUID feature bits](https://en.wikipedia.org/wiki/Bit_Manipulation_Instruction_Sets#ABM) by the same name. I think you're entirely justified in using a more human friendly name like `instruction_count` here.

Reply:
Thanks for this feedback!  I'll change the name.